tests/e2e/test.bats

```diff
@@ -8,6 +8,15 @@ ETCD_CA_CERT=/etc/kubernetes/pki/etcd/ca.crt
 ETCD_CERT=/etc/kubernetes/pki/etcd/server.crt
 ETCD_KEY=/etc/kubernetes/pki/etcd/server.key
 
+setup() {
```

Comment:
`setup()` will run before every test. I think instead we can use `setup_file()` which is run once at the beginning of the test suite. 

Ref: https://bats-core.readthedocs.io/en/latest/writing-tests.html#setup-and-teardown-pre-and-post-test-hooks

Reply:
done